src/components/Navbar/Navbar.jsx

```diff
@@ -4,7 +4,7 @@ import { useNavigate } from 'react-router';
 import {useAuth} from '../../context/auth-context'
 const Navbar = () => {
     const navigate = useNavigate();
-    const {currentUser} =useAuth();
+    const {currentUser,signout} =useAuth();
```

Comment:
You can use prettify so that your code will get auto indented, as other places there is spaces in object destructuring but here it is not.